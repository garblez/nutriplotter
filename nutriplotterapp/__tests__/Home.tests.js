import 'react-native';
import React from 'react';
//import App from '../App';
import Home from '../screens/HomeScreen'
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';


describe('App snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders home screen', async () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  })


});
