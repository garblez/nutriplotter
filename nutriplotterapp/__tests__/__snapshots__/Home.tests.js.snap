// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders home screen 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
        },
        Object {
          "bottom": 0,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 9,
                "justifyContent": "space-around",
              }
            }
          >
            <ARTSurfaceView
              style={
                Array [
                  undefined,
                  Object {
                    "height": 200,
                    "width": 200,
                  },
                ]
              }
            >
              <ARTGroup
                opacity={1}
                transform={
                  Array [
                    6.123233995736766e-17,
                    -1,
                    1,
                    6.123233995736766e-17,
                    0,
                    200,
                  ]
                }
              >
                <ARTShape
                  d={
                    Array [
                      0,
                      150,
                      100,
                      4,
                      100,
                      100,
                      50,
                      0,
                      360,
                      1,
                    ]
                  }
                  fill={null}
                  opacity={1}
                  stroke={
                    Array [
                      1,
                      1,
                      1,
                      1,
                    ]
                  }
                  strokeCap={1}
                  strokeDash={null}
                  strokeJoin={1}
                  strokeWidth={100}
                  transform={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                />
                <ARTShape
                  d={
                    Array [
                      0,
                      150,
                      100,
                      4,
                      100,
                      100,
                      50,
                      0,
                      6.283185307179586,
                      1,
                    ]
                  }
                  fill={null}
                  opacity={1}
                  stroke={
                    Array [
                      1,
                      1,
                      1,
                      1,
                    ]
                  }
                  strokeCap={1}
                  strokeDash={null}
                  strokeJoin={1}
                  strokeWidth={100}
                  transform={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                />
                <ARTShape
                  d={
                    Array [
                      0,
                      150,
                      100,
                      4,
                      100,
                      100,
                      50,
                      0,
                      3.518583772020569,
                      1,
                    ]
                  }
                  fill={null}
                  opacity={1}
                  stroke={
                    Array [
                      1,
                      1,
                      0,
                      1,
                    ]
                  }
                  strokeCap={0}
                  strokeDash={null}
                  strokeJoin={1}
                  strokeWidth={100}
                  transform={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                />
                <ARTShape
                  d={
                    Array [
                      0,
                      53.51117570558744,
                      81.59377236576609,
                      4,
                      100,
                      100,
                      50,
                      3.518583772020569,
                      4.209734155810323,
                      1,
                    ]
                  }
                  fill={null}
                  opacity={1}
                  stroke={
                    Array [
                      0,
                      0.5019607843137255,
                      0,
                      1,
                    ]
                  }
                  strokeCap={0}
                  strokeDash={null}
                  strokeJoin={1}
                  strokeWidth={100}
                  transform={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                />
                <ARTShape
                  d={
                    Array [
                      0,
                      75.91231629491423,
                      56.18466599780682,
                      4,
                      100,
                      100,
                      50,
                      4.209734155810323,
                      9.047786842338605,
                      1,
                    ]
                  }
                  fill={null}
                  opacity={1}
                  stroke={
                    Array [
                      1,
                      0.6470588235294118,
                      0,
                      1,
                    ]
                  }
                  strokeCap={0}
                  strokeDash={null}
                  strokeJoin={1}
                  strokeWidth={100}
                  transform={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                />
              </ARTGroup>
            </ARTSurfaceView>
          </View>
        </View>
      </View>
    </View>
    <Text
      style={
        Object {
          "fontSize": 24,
          "marginLeft": 24,
          "marginTop": 24,
        }
      }
    >
      Enter a food:
    </Text>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View>
        <TextInput
          allowFontScaling={true}
          onChangeText={[Function]}
          placeHolder="Enter a food"
          style={
            Object {
              "borderColor": "#111111",
              "borderWidth": 1,
              "height": "15%",
              "margin": "5%",
              "paddingHorizontal": "5%",
            }
          }
          underlineColorAndroid="transparent"
          value=""
        />
        <View
          style={
            Object {
              "height": "70%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "red",
                "textAlign": "center",
              }
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
